ui/src/authorizations/components/BucketsSelector.tsx

```diff
@@ -1,22 +1,22 @@
 // Libraries
-import React, {PureComponent, ChangeEvent} from 'react'
+import React, {PureComponent} from 'react'
 
-// Libraries
+// Components
 import BuilderCard from 'src/timeMachine/components/builderCard/BuilderCard'
-import SelectorList from 'src/timeMachine/components/SelectorList'
+import BucketsTabBody from 'src/authorizations/components/BucketsTabBody'
+import {BucketTab} from 'src/authorizations/utils/permissions'
+import BucketsTabSelector from 'src/authorizations/components/BucketsTabSelector'
 
 // Types
 import {Bucket} from '@influxdata/influx'
 import {
-  Input,
-  Button,
-  ComponentSize,
   ComponentSpacer,
   AlignItems,
   FlexDirection,
+  Button,
+  EmptyState,
 } from '@influxdata/clockface'
-import SortingHat from 'src/shared/components/sorting_hat/SortingHat'
-import FilterList from 'src/shared/components/Filter'
+import {ComponentSize} from 'src/clockface'
```

Comment:
I think you might be able to get this from @influxdata/clockface as well.

Reply:
I fixed it!